www/perl-lib/SMT/Mirror/Job.pm

```diff
@@ -426,6 +428,11 @@ sub wasNotFound()
     return $self->{RESPONSE_CODE} == 404;
 }
 
+sub getErrorMessage() {
+    my $self = shift;
+    return $self->{ERROR_MESSAGE};
+}
+
```

Comment:
The additional blank line after the inserted new subroutine at line 435. I mention it because our house Ruby style has quite strict conventions like only one line of blank space separating things.